radix-engine-interface/src/blueprints/access_controller/invocations.rs

```diff
@@ -4,36 +4,28 @@ use crate::blueprints::resource::*;
 use crate::*;
 use sbor::rust::fmt::Debug;
 
+pub const ACCESS_CONTROLLER_BLUEPRINT: &str = "AccessController";
+
 //=================================
 // Access Controller Create Global
 //=================================
 
-#[derive(Debug, Clone, Eq, PartialEq, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
-pub struct AccessControllerCreateGlobalInvocation {
-    pub controlled_asset: BucketId,
+pub const ACCESS_CONTROLLER_CREATE_GLOBAL_IDENT: &str = "create_global";
+
+#[derive(Debug, Eq, PartialEq, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
+pub struct AccessControllerCreateGlobalInput {
```

Comment:
`XyzInput` is back lol

Reply:
lol give it 2 months and `Invocation` will be back :)